modules/chanserv/list.c

```diff
@@ -56,18 +63,22 @@ parse_age(const char *s)
 	return duration;
 }
 
-static void
-process_parvarray(const struct list_option *opts, size_t optsize, int parc, char *parv[])
+static unsigned int
```

Comment:
```suggestion
static enum list_opterr
```

Since you only return enum values, the return type should be the enum.